core/services/relay/evm/mercury/transmitter.go

```diff
@@ -136,27 +141,38 @@ func NewTransmitter(lggr logger.Logger, cfgTracker ConfigTracker, rpcClient wsrp
 
 func (mt *mercuryTransmitter) Start(ctx context.Context) (err error) {
 	return mt.StartOnce("MercuryTransmitter", func() error {
+		mt.lggr.Debugw("Loading transmit requests from database")
+		transmissions, err := mt.persistenceManager.Load(ctx)
+		if err != nil {
+			return err
+		}
+		mt.queue = NewTransmitQueue(mt.lggr, mt.feedIDHex, maxTransmitQueueSize, transmissions, mt.persistenceManager)
+
 		if err := mt.rpcClient.Start(ctx); err != nil {
 			return err
 		}
 		if err := mt.queue.Start(ctx); err != nil {
 			return err
 		}
 		mt.wg.Add(1)
-		go mt.runloop()
+		go mt.runQueueLoop()
 		return nil
 	})
 }
 
 func (mt *mercuryTransmitter) Close() error {
 	return mt.StopOnce("MercuryTransmitter", func() error {
-		mt.queue.Close()
+		// Stop runQueueLoop before stopping its dependencies
 		close(mt.stopCh)
 		mt.wg.Wait()
+		_ = mt.queue.Close()              // Close always returns nil
```

Comment:
I'd suggest returning the error anyway on the slim chance that in the future these methods are modified such that they might return an error